### PR TITLE
Added functional Galveston College scraper

### PR DESCRIPTION
**Pull request by HarrisonGregg**

### CODE REVIEW

File: osp_scraper/spiders/gc.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                        "subject": subject
+                    }
+                    yield scrapy.FormRequest(
+                        get_syllabi_url, 
+                        method="POST",
+                        formdata=formdata,
+                        meta=formdata,
+                        callback=get_courses
+                    )
+                    return
```
Comment (jodizzle) [inline osp_scraper/spiders/gc.py]: Is there a reason for this `return`?

File: osp_scraper/spiders/gc.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                        "course": response.meta["course"], 
+                        "section": section
+                    }
+                    yield scrapy.FormRequest(
+                        get_syllabi_url, 
+                        method="POST",
+                        formdata=formdata,
+                        meta={
+                            "depth": 1,
+                            "hops_from_seed": 1,
```
Comment (jodizzle) [inline osp_scraper/spiders/gc.py]: Should both `depth` and `hops_from_seed` be 1 here?

File: osp_scraper/spiders/gc.py (hunk 1 above)
Comment (HarrisonGregg) [inline osp_scraper/spiders/gc.py]: Nope. Good catch.

File: osp_scraper/spiders/gc.py (hunk 2 above)
Comment (HarrisonGregg) [inline osp_scraper/spiders/gc.py]: Do you have reason to believe they shouldn't be? I have literally no idea what they are actual used for.

Comment (jodizzle) [inline osp_scraper/spiders/gc.py]: I think you should probably set the `depth` and `hops_from_seed` to 1 in `get_subjects`, and increment each by 1 for each successive post request.

**Comment by HarrisonGregg** [comment]
@jodizzle I was able to refactor it to simplify it even further because of how from_response handles things.  How does it look?

File: osp_scraper/spiders/gc.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                },
+                callback=self.parse_for_sections
+            )
+
+    def parse_for_sections(self, response):
+        term = response.css("#term::attr(value)").extract_first()
+        subject = response.css("#subject::attr(value)").extract_first()
+        course = response.css("#course::attr(value)").extract_first()
+        for option in response.css("#section option"):
+            section = option.css("option::attr(value)").extract_first()
```
Comment (jodizzle) [inline osp_scraper/spiders/gc.py]: Can't you just use `::attr(value)`

**Comment by jodizzle** [comment]
This looks very clean now.

**Pull request merged**